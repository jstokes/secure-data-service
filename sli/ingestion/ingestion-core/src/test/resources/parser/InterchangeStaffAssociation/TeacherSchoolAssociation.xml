<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStaffAssociation xmlns:sli="http://slc-sli/ed-org/0.1"
 xmlns="http://ed-fi.org/0100"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="http://ed-fi.org/0100 file:/C:/beta/sli/sli/edfi-schema/src/main/resources/edfiXsd-SLI/SLI-Interchange-StaffAssociation.xsd">
    <TeacherSchoolAssociation id="ID015">
        <TeacherReference>
            <StaffIdentity>
                <StaffUniqueStateId>HjE3Gyf</StaffUniqueStateId>
            </StaffIdentity>
        </TeacherReference>
        <SchoolReference>
            <EducationalOrgIdentity>
                <StateOrganizationId>n</StateOrganizationId>
            </EducationalOrgIdentity>
        </SchoolReference>
        <ProgramAssignment>Regular Education</ProgramAssignment>
        <InstructionalGradeLevels>
            <GradeLevel>Adult Education</GradeLevel>
            <GradeLevel>Tenth grade</GradeLevel>
            <GradeLevel>Adult Education</GradeLevel>
            <GradeLevel>Postsecondary</GradeLevel>
            <GradeLevel>Postsecondary</GradeLevel>
            <GradeLevel>Second grade</GradeLevel>
        </InstructionalGradeLevels>
        <AcademicSubjects>
            <AcademicSubject>Writing</AcademicSubject>
            <AcademicSubject>Architecture and Construction</AcademicSubject>
            <AcademicSubject>English Language and Literature</AcademicSubject>
            <AcademicSubject>Architecture and Construction</AcademicSubject>
            <AcademicSubject>Social Studies</AcademicSubject>
            <AcademicSubject>Life and Physical Sciences</AcademicSubject>
        </AcademicSubjects>
    </TeacherSchoolAssociation>
</InterchangeStaffAssociation>
