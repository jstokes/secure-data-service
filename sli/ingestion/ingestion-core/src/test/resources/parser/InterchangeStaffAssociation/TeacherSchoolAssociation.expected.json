{
  "@id":"ID015",
  "TeacherReference":{
    "StaffIdentity":{
      "StaffUniqueStateId":{"_value":"HjE3Gyf"}
    }
  },
  "SchoolReference":{
    "EducationalOrgIdentity":{
      "StateOrganizationId":{"_value":"n"}
    }
  },
  "ProgramAssignment":{"_value":"Regular Education"},
  "InstructionalGradeLevels":{
    "GradeLevel":[
      {"_value":"Adult Education"},
      {"_value":"Tenth grade"},
      {"_value":"Adult Education"},
      {"_value":"Postsecondary"},
      {"_value":"Postsecondary"},
      {"_value":"Second grade"}
    ]
  },
  "AcademicSubjects":{
    "AcademicSubject":[
      {"_value":"Writing"},
      {"_value":"Architecture and Construction"},
      {"_value":"English Language and Literature"},
      {"_value":"Architecture and Construction"},
      {"_value":"Social Studies"},
      {"_value":"Life and Physical Sciences"}
    ]
  }
}